bundles/org.openhab.binding.dreamscreen/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dreamscreen"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<thing-type id="dreamScreen">
+		<label>DreamScreen TV</label>
+		<description>Controls the DreamScreen TV</description>
+
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="input" typeId="input" />
+			<channel id="mode" typeId="mode" />
+			<channel id="scene" typeId="scene" />
+			<channel id="color" typeId="color" />
+		</channels>
+
+		<config-description>
+			<parameter name="name" type="text" required="true">
+				<label>Name</label>
+				<description>The name for the DreamScreen</description>
+				<default>DreamScreen 4K</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Turns the power to the DreamScreen on/off</description>
+	</channel-type>
+
+	<channel-type id="mode">
+		<item-type>Number</item-type>
+		<label>Mode</label>
+		<description>Switch the display mode</description>
+		<state>
+			<options>
+				<option value="0">Video</option>
+				<option value="1">Music</option>
+				<option value="2">Ambient</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="scene">
+		<item-type>Number</item-type>
```

Comment:
Channels with options should be String.
```suggestion
		<item-type>String</item-type>
```
The numbers seem unrelated to the device itself but used as index. Maybe use the name of the enum (either in lower case) as value and then use `valueOf(` method to get the specific enum. It makes it also a bit more logical for the user.

Reply:
changed